StepLang.CLI/example-format-config.json

```diff
@@ -0,0 +1,6 @@
+﻿{
+  "encoding": "utf-8",
+  "lineEndings": "lf",
+  "indentation": "tab",
+  "indentationSize": 4
```

Comment:
if it's four chars, it's _not_ a tab, right?

Reply:
The file no longer exists